framework/Models/PostProcessors/Factory.py

```diff
@@ -39,6 +39,7 @@
 from .ParetoFrontierPostProcessor import ParetoFrontier
 from .MCSimporter import MCSImporter
 from .EconomicRatio import EconomicRatio
+from .RiskMeasuresDiscrete import RiskMeasuresDiscrete
```

Comment:
I am confused about this "double" import (both in the ```__init__``` and ```Factory.py```).

See my PR about the validation gate, the double import is not needed as long as the imports are in the ```Factory.py```. This is valid for all the Factories that have been "Restructured" with PR  #1480

Reply:
That's good to know, we can do a quick clean up in another PR, since I do not see the double imports get removed.

Reply:
yes but we can remove it from here

Reply:
the ```__init__``` got cleaned up in previous PR.